res/shaders/rendergraph/README

```diff
@@ -0,0 +1,9 @@
+Generate the GLSL shaders from the spirv shaders by running
+
+generate_shaders_gl.pl
+
+(Make sure qsb and spirv commands are in your path. E.g:
+export PATH=$PATH:~/VulkanSDK/1.3.283.0/macOS/bin:~/Qt/6.7.2/macos/bin
+)
+
+Since Qt 6.6 we should be able to access this programmatically with QShader, but for now I do it manually
```

Comment:
The GLSL shader can be retrieved from the qsb bundles in runtime: https://doc.qt.io/qt-6/qshader.html , and that would be the way to go. But QShader exists since Qt 6.6 only.

Reply:
I could add the use of QShader to the code in an if qt version >= 6.6 block, so it is clear that this solution is only temporary until we move to a more recent Qt.

Reply:
I see, but that still leaves in the qsb artifacts in the tree. Those could be generated during build-time right?

Reply:
Done!